src/components/tile/components.test-pw.tsx

```diff
@@ -77,3 +85,39 @@ export const TileComponentWithFalsyChildren = () => {
     </Tile>
   );
 };
+
+export const ResponsiveTile = () => {
+  return (
+    <Tile m={0} py={0}>
+      <ResponsiveTileContainer>
+        <ResponsiveCell py={2}>Test Body One</ResponsiveCell>
+        <ResponsiveCell py={2}>Test Body Two</ResponsiveCell>
+        <ResponsiveCell py={2}>
+          Test Body Three With a very very long text
+        </ResponsiveCell>
+      </ResponsiveTileContainer>
+    </Tile>
+  );
+};
+
+export const ResponsiveTileContainerWithFalsyChildren = () => {
+  return (
+    <>
+      <ResponsiveTileContainer />
+      <ResponsiveTileContainer>Foo</ResponsiveTileContainer>
+      <ResponsiveTileContainer>{null}</ResponsiveTileContainer>
+      {false && <ResponsiveTileContainer>Bar</ResponsiveTileContainer>}
+    </>
+  );
+};
+
+export const ResponsiveCellWithFalsyChildren = () => {
```

Comment:
Comment: I think this can be a test story or we can just rely on unit/ Ci tests to capture this

Reply:
Done in UT